### PR TITLE
Upd README

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +25,28 @@ There is no documentation for the different SDKs except the dedicated READMEs. T
 
 If you want to build an integration for MeiliSearch, you are more than welcome to and we would be happy to help you :)
 
+We are proud to say that some of our libraries was done and still maintained by external contributors!
```
Comment (tpayet) [inline README.md:28]:
```suggestion
We are proud that some of our libraries were developed and are still maintained by external contributors ❤️
```

File: README.md [hunk 2]
```diff
@@ -24,12 +25,28 @@ There is no documentation for the different SDKs except the dedicated READMEs. T
...
+
 Here are some guidelines that you should follow though, if we want to guarantee consistency across the differents integrations and technical quality for the different MeiliSearch users.
 
-#### About testing
+#### README
+
+All the README should follow the same patterns. This repository should hold the README template and at some point, we might want to automatically update all the readmes from here.
```
Comment (tpayet) [inline README.md:34]:
```suggestion
All READMEs should follow the same pattern. This repository should hold the README template and at some point, we might want to automatically update all the READMEs from here.
```

File: README.md [hunk 3]
```diff
@@ -24,12 +25,28 @@ There is no documentation for the different SDKs except the dedicated READMEs. T
...
+
+Since we don't provide an official template in this repository, here are the minimum part that the README must contain:
+
+- A well-explained `Getting Started` part: the user should be able to copy/paste the code and command examples and everything should work as expected.
+- Basic code examples
```
Comment (tpayet) [inline README.md:39]:
```suggestion
- Basic code examples for the most common actions
```

File: README.md [hunk 4]
```diff
@@ -24,12 +25,28 @@ There is no documentation for the different SDKs except the dedicated READMEs. T
...
+We recommend to check out the READMEs of our SDKs repositories 🙂
+
+#### Tests
 
 If you want to start writing tests you can read what the [PHP SDK](https://github.com/meilisearch/meilisearch-php) is testing to get an overview of minimum viable tests that your SDK should support. The [JS](https://github.com/meilisearch/meilisearch-js) and the [Ruby](https://github.com/meilisearch/meilisearch-ruby) SDKs are also good examples of deeper tests.
 
-#### README
+#### Linter and Style
+
+A linter should be added to optimize contributions.
```
Comment (tpayet) [inline README.md:50]:
```suggestion
A linter should be added to ease contributions and maintainers work.
```

File: README.md [hunk 5]
```diff
@@ -24,12 +25,28 @@ There is no documentation for the different SDKs except the dedicated READMEs. T
...
 
-All the README should follow the same patterns. This repository should hold the README template and at some point, we might want to automatically update all the readmes from here. 
+We also recommend to add a `.editorconfig` file so that everyone can follow minimal style rules.
```
Comment (tpayet) [inline README.md:52]:
```suggestion
We also recommend to add a `.editorconfig` file so that everyone can follow minimal styling rules.
```

**Comment by bidoubiwa** [comment]
Javascript is missing in the list !

**Pull request merged**